puppet/zulip/manifests/app_frontend_base.pp

```diff
@@ -163,6 +163,7 @@
     'missedmessage_emails',
     'missedmessage_mobile_notifications',
     'outgoing_webhooks',
+    'register_push_device_to_bouncer',
```

Comment:
Does this need a new queue? I would think that the `missedmessage_mobile_notifications` might be appropriate for it.

Reply:
Removed this new queue.

Earlier, used a new queue just for logical separation. `missedmessage_mobile_notifications` for the missed messages; `'register_push_device_to_bouncer'` for queueing registration requests.